From b3b7951d991c7ef16834e3946742bddd4b3b848e Mon Sep 17 00:00:00 2001
From: Kondapally Kalyan <kalyan.kondapally@intel.com>
Date: Mon, 31 Mar 2014 08:59:10 -0300
Subject: [PATCH] EGL specific changes for Wayland.

OffScreen-rendering support for use cases like WebGL and Canvas2D. This should
be enabled in upstream but needs some cross platform support before it can be
done. More info: https://codereview.chromium.org/49533003/
---
 ui/gl/gl_surface_ozone.cc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/gl/gl_surface_ozone.cc b/ui/gl/gl_surface_ozone.cc
index 5cf70c7..fb844e9 100644
--- a/ui/gl/gl_surface_ozone.cc
+++ b/ui/gl/gl_surface_ozone.cc
@@ -82,7 +82,7 @@ scoped_refptr<GLSurface> GLSurface::CreateOffscreenGLSurface(
     case kGLImplementationEGLGLES2: {
       scoped_refptr<GLSurface> surface;
       if (GLSurfaceEGL::IsEGLSurfacelessContextSupported() &&
-          (size.width() == 0 && size.height() == 0)) {
+          (size.width() == 1 && size.height() == 1)) {
         surface = new SurfacelessEGL(size);
       } else
         surface = new PbufferGLSurfaceEGL(size);
-- 
1.7.9.5

